incubator/artifactory/values.yaml

```diff
@@ -0,0 +1,29 @@
+# Default values for Artifactory.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+Name: artifactory
+Component: "Artifactory"
+
+## Uncomment following line if you want to run Artifactory-Pro
+#image: "docker.bintray.io/jfrog/artifactory-pro"
+imageTag: "4.14.1"
+imagePullPolicy: "Always"
+replicaCount: 1
+httpPort: 8081
+## Kubernetes configuration
+## For minikube, set this to NodePort, elsewhere use LoadBalancer
+##
+ServiceType: LoadBalancer
```

Comment:
Lets make this ClusterIP by default since the password isnt randomly generated by default.